certified-connectors/WithoutWire Inventory Platform/apiProperties.json

```diff
@@ -0,0 +1,44 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "baseurl": {
+        "type": "string",
+        "uiDefinition": {
+          "displayName": "Base Url",
+          "description": "Base Url (https://api.withoutwire.com/)",
+          "tooltip": "Provide bas url",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": true,
+            "required": "true"
+          }
+        }
+      },
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "API Key",
+          "description": "The API Key for this api",
+          "tooltip": "Provide your API Key",
+          "constraints": {
+            "tabIndex": 3,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#ffffff",
+    "capabilities": [],
+    "policyTemplateInstances": [
+      {
+        "templateId": "dynamichosturl",
+        "title": "Set URL",
+        "parameters": {
+          "x-ms-apimTemplateParameter.urlTemplate": "@connectionParameters('baseurl')"
+        }
+      }
+    ],
+    "publisher": "WithoutWire Inventory Sciences"
```

Comment:
@PavloNikulinEnavate Can you please add `stackOwner` property as well.

Documentation - [Adding Metadata](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#step-4-add-metadata)